README.md

```diff
@@ -214,27 +215,55 @@ public:
 ## Configuration
 
 The behavior of this library is configured by one preprocessor symbol,
-`ADOBE_CONTRACT_VIOLATION`.  It can be defined to one of three
+`ADOBE_CONTRACT_VIOLATION`. It can be defined to one of three
 strings, or be left undefined, in which case it defaults to `verbose`.
 
 - `ADOBE_CONTRACT_VIOLATION=verbose`: as much information as possible
   is collected from the site of a detected contract violation and
   reported to the standard error stream before `std::terminate()`
   is called.
-  
 - `ADOBE_CONTRACT_VIOLATION=lightweight`: When a contract violation is
-  detected, `std::terminate()` is invoked.  Aside from
+  detected, `std::terminate()` is invoked. Aside from
   code to check the condition and call `terminate`, none of the
   arguments to a contract checking macro generates any code or data.
 
 - `ADOBE_CONTRACT_VIOLATION=unsafe`: Contract checking macros have no
-  effect and generate no code or data.  Not recommended for general
+  effect and generate no code or data. Not recommended for general
   use, but can be useful for measuring the overall performance impact
   of checking in a program.
 
-Except in `unsafe` mode, a failed check ultimately calls
+- `ADOBE_CONTRACT_VIOLATION=custom_verbose`: When a contract violation is detected, a custom handler
+  is invoked. The client must define the handler at global scope with the signature:
```

Comment:
This _sounds_ like it's saying you have to write it in the global namespace, and then you look closer and see that it's in `adobe::`, and go, “nonsense, I can do it like this:”

```c++
namespace adobe {
  void adobe_contract_violation_verbose(...
}
```

This is not a big deal, but a phrasing that doesn't cause this mental hiccup would be better if we can find one.

Reply:
It's not in the `adobe` namespace.

Reply:
Misreading on my part as noted elsewhere.